Index: rsvp.module
===================================================================
--- rsvp.module	(revision 575)
+++ rsvp.module	(working copy)
@@ -2178,7 +2178,7 @@
     '#rows' => 3,
     '#prefix' => '<div class="rsvp_form_content rsvp_add_guests_form_content rsvp_color_inner">',
     '#suffix' => '</div>',
-    '#description' => t('Enter email addresses or !site usernames that you want to add as guests each in a separate line.', array('!site'=>variable_get("site_name", "drupal"))),
+    '#description' => t('Enter email addresses or !site usernames that you want to add as guests. Each in a separate line.', array('!site'=>variable_get("site_name", "drupal"))),
     '#weight' => 55,
   );
   
