Index: node_comment_form.inc
===================================================================
--- node_comment_form.inc	(revision 252)
+++ node_comment_form.inc	(working copy)
@@ -19,6 +19,10 @@
 
 function ctools_node_comment_form_content_type_render($subtype, $conf, $panel_args, $context) {
   $node = isset($context->data) ? drupal_clone($context->data) : NULL;
+	
+  if (empty($node) && @$panel_args[0])
+    $node = node_load (array('uid' => $panel_args[0], 'type' => 'profile'));
+
   $block = new stdClass();
   $block->module = 'comments';
   $block->delta  = $node->nid;
Index: node_comments.inc
===================================================================
--- node_comments.inc	(revision 252)
+++ node_comments.inc	(working copy)
@@ -24,6 +24,10 @@
 
 function ctools_node_comments_content_type_render($subtype, $conf, $panel_args, $context) {
   $node = isset($context->data) ? drupal_clone($context->data) : NULL;
+	
+  if (empty($node) && @$panel_args[0])
+    $node = node_load (array('uid' => $panel_args[0], 'type' => 'profile'));
+
   $block = new stdClass();
   $block->module = 'comments';
   $block->delta  = $node->nid;
