Index: quickbar.module
===================================================================
--- quickbar.module	(revision 369)
+++ quickbar.module	(working copy)
@@ -303,6 +303,9 @@
       if (quickbar_in_active_trail($item['link']['href'])) {
         $class .= ' active-trail';
       }
+			
+			$class .= ' ' . preg_replace ("/[^a-zA-Z]/", "_", $item['link']['title']);
+			
       // Keyed with the unique mlid to generate classes in theme_links().
       $links['menu-'. $item['link']['mlid'] . $class] = $l;
     }
