Index: location_testcase.php
===================================================================
--- location_testcase.php	(revision 252)
+++ location_testcase.php	(working copy)
@@ -16,10 +16,10 @@
       $lower = $test - $epsilon;
       $upper = $test + $epsilon;
       if ($result[$k] < $lower || $result[$k] > $upper) {
-        $this->_assert('fail', $message ? $message : t('Value deviates by @amt, which is more than @maxdev.', array('@amt' => abs($test - $result[$k]), '@maxdev' => $epsilon)), $group);
+        $this->assert('fail', $message ? $message : t('Value deviates by @amt, which is more than @maxdev.', array('@amt' => abs($test - $result[$k]), '@maxdev' => $epsilon)), $group);
       }
       else {
-        $this->_assert('pass', $message ? $message : t('Value within expected margin.'), $group);
+        $this->assert('pass', $message ? $message : t('Value within expected margin.'), $group);
       }
     }
   }
