Index: views_groupby.test
===================================================================
--- views_groupby.test	(revision 229)
+++ views_groupby.test	(working copy)
@@ -252,6 +252,10 @@
     $view->set_display('default');
     $output = $view->execute_display();
 
+    $this->assertTrue((bool)$view->query, 'View has query');
+    if (!$view->query)
+      return;
+
     $cache_info = $view->query->get_cache_info();
 
     $this->assertTrue(strpos($cache_info['final_query'], 'GROUP BY'), t('Make sure that GROUP BY is in the query'));
