Index: urlicon.module
===================================================================
--- urlicon.module	(revision 608)
+++ urlicon.module	(working copy)
@@ -142,7 +142,11 @@
   $domain = explode('.', $url['host']);
   $domain = check_url(str_replace('.', '_', $url['host']));
 
+		
   //check if favicon exists locally
+	if (strpos ($_SERVER ['HTTP_HOST'], $url['host']) !== false || strpos ($url['host'], $_SERVER ['HTTP_HOST']) !== false)
+		return theme('urlicon', $match[3], $favicon, $match[1], array('alt' => '', 'title' => t('favicon')));
+	else
   if ($url['host'] AND !file_exists($dir .'/'. $domain .'.ico')) {
   
     //check for favicon in metatags
